src/Microsoft.Performance.SDK/Processing/IProcessingSource.cs

```diff
@@ -43,6 +44,11 @@ public interface IProcessingSource
         /// </summary>
         IEnumerable<Option> CommandLineOptions { get; }
 
+        /// <summary>
+        ///     Gets the plugin options that are supported by this <see cref="IProcessingSource"/>.
+        /// </summary>
+        IEnumerable<PluginOption> PluginOptions { get; }
+
```

Comment:
breaking change? do we care?

Reply:
Breaking change, but we decided when we did previous features that we're OK breaking people who directly implement IProcessingSource. For example, we broke this interface when we added the `IDataSourceGroup` functionality.